src/ontology/mondo-edit.obo

```diff
@@ -452929,9 +452928,11 @@ xref: MEDGEN:1382737 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:79091 {source="GARD:0009494", source="MONDO:equivalentTo"}
 xref: SCTID:724349009 {source="MONDO:equivalentTo"}
 xref: UMLS:C4510610 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1382737"}
-is_a: MONDO:0011577 ! myopathy, proximal, and ophthalmoplegia
+is_a: MONDO:0011577 {source="OMIM:605637"} ! myopathy, proximal, and ophthalmoplegia
```

Comment:
Same as above, OMIM is not equivalent to this term, therefore it cannot be used as a source. 
I think the orphanet ID would work here.